rust/private/rust.bzl

```diff
@@ -885,7 +885,10 @@ _rust_binary_attrs = {
         cfg = "exec",
         allow_single_file = True,
     ),
-    "out_binary": attr.bool(),
+    "out_binary": attr.bool(
+        doc = "Force a target, regardless of it's `crate_type`, to always mark the file as executable.",
```

Comment:
WDYT about mentioning that this is really only for wasm, how exactly it is supposed to be used in wasm, and that it will be removed in the doc?

Reply:
How's this?